src/Silverback.Integration/Messaging/IEndpoint.cs

```diff
@@ -49,6 +50,12 @@ public interface IEndpoint
         /// </summary>
         string? FriendlyName { get; }
 
+        /// <summary>
+        ///     Gets or sets the validation mode. This option can be used to determines if validation has to be performed and if
```

Comment:
`This option can be used to specify if the messages have to be validated and whether an exception must be thrown if the message is not valid.`